src/sth_logger.js

```diff
@@ -11,21 +11,17 @@
   var sthConfig;
 
   var LOG_LEVEL = {
-    trace: 'trace',
-    info: 'info',
     debug: 'debug',
+    info: 'info',
     warn: 'warn',
-    error: 'error',
-    fatal: 'fatal'
```

Comment:
Umm, Cygnus uses the `fatal` one xD

I'll check it later, in the meantime, NTC